static/app/views/alerts/rules/metric/details/metricChartOption.tsx

```diff
@@ -346,12 +347,20 @@ export function getMetricAlertChartOption({
     maxThresholdValue = Math.max(maxThresholdValue, rule.resolveThreshold);
   }
 
+  const crashFreeMax =
```

Comment:
can we use the helper function `isCrashFreeAlert` here and pass the rule.dataset